include/nodes/internal/ConnectionStyle.hpp

```diff
@@ -14,25 +14,25 @@ class NODE_EDITOR_PUBLIC ConnectionStyle : public Style
 
   ConnectionStyle();
 
-  ConnectionStyle(QString jsonText);
+  ConnectionStyle(const QString& jsonText);
 
 public:
 
-  static void setConnectionStyle(QString jsonText);
+  static void setConnectionStyle(const QString& jsonText);
 
 private:
 
-  void loadJsonText(QString jsonText) override;
+  void loadJsonText(const QString& jsonText) override;
```

Comment:
This wasn't really necessary due to https://doc.qt.io/qt-5/implicit-sharing.html

Reply:
function-args-by-ref

Warns when you should be passing by const-ref. Types with sizeof > 16 bytes [1] or types which are not trivially-copyable [2] or not trivially-destructible [3] should be passed by ref. A rule of thumb is that if passing by value would trigger copy-ctor and/or dtor then pass by ref instead.